src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -47,6 +50,7 @@ public static void FlushPayload()
                 if (payload != null)
                 {
                     Console.WriteLine(i);
+                    Console.WriteLine(_fileIndices[i]);
```

Comment:
Consider adding a prefix for readability:   `Console.WriteLine($""[File:](File:) {_fileIndices[i]}"");`
Same suggestion for original WriteLine (one line above).